package.json

```diff
@@ -4,23 +4,34 @@
   "description": "React implementation of OneTrust consent banner integrated with Google Analytics signals.",
   "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
   },
   "keywords": [],
   "author": "jaydonkrooss",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/jaydonkrooss/react-ga-onetrust-consent.git"
+    "url": "https://github.com/jaydonkrooss/react-ga-onetrust-consent.git"
```

Comment:
Don't forget to update with reference to TL's repo.

Reply:
nice catch. should now be up to date with tl organization name.